components/python/jinja2/Makefile

```diff
@@ -11,11 +11,12 @@
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
 # Copyright 2015, PALO Richard.
+# Copyright 2017 Gary Mills
 #
 
 include ../../../make-rules/shared-macros.mk
 
-PATH=/usr/bin:/usr/gnu/bin:/usr/sbin
+PATH = $(PATH.ill):/usr/sbin
```

Comment:
I used PATH.ill because I wanted to keep it the same length as PATH.gnu .  I could change it to PATH.illumos or anything else, but it has to match the symbol already defined in make-rules/shared-macros.mk .  This would need another commit.  I'd also have to change about 50 component Makefile to match it, and feed them through in separate commits.  If we want to change it, we have to decide right now.